ethpm/contract.py

```diff
@@ -15,11 +15,11 @@ class LinkableContract(Contract):
     contract factories with link references in their package's manifest.
     """
 
-    unlinked_references: Tuple[Dict[str, Any]] = None
-    linked_references: Tuple[Dict[str, Any]] = None
+    unlinked_references: Optional[Tuple[Dict[str, Any]]] = None
```

Comment:
Should the inner dict be mutable?  If not maybe convert to `FrozenDict` at which point you could assign `(frozendict(),)` as a class property since it's immutable and then not have to worry about the setting of the default value in the constructor.